src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -137,6 +161,15 @@ public static boolean deleteFolder(final File file)
         return false;
     }
 
+    public static boolean deleteFile(final File file)
```

Comment:
Can you remove if from your PR then please? It just doenst appear to have any functionality.